test/e2e/angular-app/extra-webpack.config.js

```diff
@@ -0,0 +1,22 @@
+require('../env').config();
+
+const webpack = require('webpack');
+const env = {};
+
+// List of environment variables made available to the app
+[
+  'PORT',
+  'WIDGET_TEST_SERVER',
+  'WIDGET_CLIENT_ID'
+].forEach(function (key) {
+  env[key] = JSON.stringify(process.env[key]);
```

Comment:
why need `stringify`?

Reply:
If the value is a string it must be "quoted" or we will get syntax error when webpack substitutes it in the source.